.github/workflows/tests.yml

```diff
@@ -50,4 +50,4 @@ jobs:
           pip install -e .[test]
       - name: Test with pytest
         run: |
-          make test
+          make test_custom_model
```

Comment:
Could you please undo this change and the change to the `Makefile`? If it's to speed up your local testing, you can run `python -m pytest tests/ -k LayerNorm` and it should only run your new tests.